cassiohcore/Commands/LaEle.ts

```diff
@@ -0,0 +1,30 @@
+import { CommandModel } from "../Modal/CommandModel";
+import { IMessage_format } from "../Modal/MessageModel";
+
+export class LaEleCommand extends CommandModel {
+    protected _key: string = "laele";
+    protected _name: string = "/laele";
+    protected _alias: string | undefined = undefined;
+    protected _options?: string[] | undefined = undefined;
+    protected _description: string = "LÁ ELE!";
+    protected _access_level:number = 0;
```

Comment:
By default, all users access code for _access_level is 4, but this is not used as this moment so pog.